docs/Troubleshooting.md

```diff
@@ -169,9 +171,49 @@ This is a known issue an occurs on first upload of Linux data into an OMS worksp
 * If using a proxy, check proxy troubleshooting steps aboce
 * In some cases, when the OMS Agent for Linux cannot talk to the OMS Service, data on the Agent is backed up to the full buffer size: 50 MB. The OMS Agent for Linux should be restarted by running the following command `service omsagent restart` or `systemctl restart omsagent`.
  * **Note:** This issue is fixed in Agent version >= 1.1.0-28
+
+### My portal side configuration for (Syslog/Linux Performance Counter) is not being applied
+#### Probable Causes
+* The OMS Agent for Linux Configuration Agent has not picked up the latest portal side configuration
+* The changed settings in the portal were not applied
+
+#### Resolutions
+
+**Background:** `omsconfig` is the OMS Agent for Linux configuration agent that looks for new portal side configuration every 5 minutes. This configuration is then applied to the OMS Agent for Linux configuration files located at /etc/opt/microsoft/omsagent/conf/omsagent.conf. 
+
+* In some cases the OMS Agent for Linux configuration agent might not be able to communicate with the portal configuration service resulting in latest configuration not being applied.
+ * Check that the `omsconfig` agent is installed
+  * `dpkg --list omsconfig` or `rpm -qi omsconfig`
+  * If not installed, reinstall the latest version of the OMS Agent for Linux
+
+* Check that the `omsconfig` agent can communicate with the OMS Portal Service
+  * Run the following command `sudo su omsagent -c 'python /opt/microsoft/omsconfig/Scripts/GetDscConfiguration.py'`
+   * This command returns the Configuration that agent sees from the portal including Syslog settings, Linux Performance Counters, and Custom Logs
+   * If this command fails try reinstalling the latest version of the OMS Agent for Linux, otherwise please report a GitHub Issue with the log from `/var/opt/microsoft/omsconfig/omsconfig.log`
+
+
+### I'm not seeing my Custom Log Data in the OMS Potal
+#### Probable Causes
+* Onboarding to OMS Service failed
+* The setting "Apply the following configuration to my Linux Servers" has not been check marked
+* omsconfig has not picked up the latest Custom Log from the portal
+* OMS Agent for Linux user `omsagent` is unable to access the Custom Log due to permissions or not being found
+ * `[DATETIME] [warn]: file not found. Continuing without tailing it.`
+ * `[DATETIME] [error]: file not accessible by omsagent.`
+* Known Issue with Race Condition fixed in OMS Agent for Linux version 1.1.0-217
+
+####Resolutions
+* Check if onboarding the OMS Service was successful by checking if the following file exists: `/etc/opt/microsoft/omsagent/conf/omsadmin.conf`
+ * Re-onboard using the omsadmin.sh command line [instructions](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/OMS-Agent-for-Linux.md#onboarding-using-the-command-line)
+* In the OMS Portal under Settings ensure that the following checkbox is checked ![](pictures/CustomLogLinuxEnabled.png?raw=true)
+
```

Comment:
Please also mention about granting read permission on the target file to omsagent.